pkg/preflights/host-preflight.yaml

```
@@ -370,6 +424,342 @@ spec:
+    - tcpPortStatus:
+        checkName: Kube API Server Port Availability
+        collectorName: Kube API Server Port
+        outcomes:
+          - fail:
+              when: "connection-refused"
+              message: Port 6443/TCP is required, but the connection was refused. This may indicate that the connection is being redirected to somewhere else on your network (perhaps a proxy configuration).
+          - fail:
+              when: "address-in-use"
+              message: Port 6443/TCP is required, but another process is already listening on it. Relocate the conflicting process to continue.
+          - fail:
+              when: "connection-timeout"
+              message: Port 6443/TCP is required, but the connection timed out. Ensure that your firewall doesn't block port 6443/TCP.
+          - fail:
+              when: "error"
+              message: Port 6443/TCP is required, but an unexpected error occurred when trying to connect to it
+          - pass:
+              when: "connected"
+              message: Port 6443/TCP is available
+          - error:
+              message: Port 6443/TCP is required, but an unexpected error occurred when trying to connect to it
+    - tcpPortStatus:
+        checkName: Envoy Port Availability
+        collectorName: Envoy Port
+        outcomes:
+          - fail:
+              when: "connection-refused"
+              message: Port 7443/TCP is required, but the connection was refused. This may indicate that the connection is being redirected to somewhere else on your network (perhaps a proxy configuration).
+          - fail:
+              when: "address-in-use"
+              message: Port 7443/TCP is required, but another process is already listening on it. Relocate the conflicting process to continue.
+          - fail:
+              when: "connection-timeout"
+              message: Port 7443/TCP is required, but the connection timed out. Ensure that your firewall doesn't block port 7443/TCP.
+          - fail:
+              when: "error"
+              message: Port 7443/TCP is required, but an unexpected error occurred when trying to connect to it
+          - pass:
+              when: "connected"
+              message: Port 7443/TCP is available
+          - error:
+              message: Port 7443/TCP is required, but an unexpected error occurred when trying to connect to it
+    - tcpPortStatus:
+        checkName: Kotsadm Node Port Availability
+        collectorName: Kotsadm Node Port
+        outcomes:
+          - fail:
+              when: "connection-refused"
+              message: Port 30000/TCP is required, but the connection was refused. This may indicate that the connection is being redirected to somewhere else on your network (perhaps a proxy configuration).
+          - fail:
+              when: "address-in-use"
+              message: Port 30000/TCP is required, but another process is already listening on it. Relocate the conflicting process to continue.
```

When we merge Ethan's port relocation changes, we can update this:
```suggestion
              message: Port 30000/TCP is required, but another process is already listening on it. To proceed, relocate the conflicting process or specify an available port using --admin-console-port.
```